### ISSUE TITLE
Resizing images

### ISSUE THREAD

**Issue by geaplanet** [jmoenig/Snap--Build-Your-Own-Blocks#692]
The editor of costumes (PNG or JPEG) has a poor shrink/grow function, quickly images get blur. On the contrary, resizing with blocks  has a much better result.

**Comment by cycomachead** [comment]
When you are trying to resize images -- are you doing it only 1 using the costume editor, or multiple times?

The two work slightly differently, but the results shouldn't be all that different. Could you share a few more details or images?

**Comment by studej** [comment]
It is really like that. I can confirm @geaplanet words.

**Comment by deader69** [comment]
Shrink is a 'lossy' transformation, inside the paint program. That's a good thing, because I'm pretty sure it reduces data size.

You can also keep the art at it's original resolution, and shrink it at run time with the 'set/change size' Looks blocks.

They're both good techniques, each with pros/cons. 
I wouldn't consider it a bug.

I teach kids to filter their google image searches by size, so they're not loading HUGE bitmaps.

**Comment by geaplanet** [comment]
I don't think it reduces data size, simply is a brutal lossy transformation. It should be work equal set/change size blocks. Now is useless and the results are horrible even if you only want reduce data size.

**Comment by introlinux** [comment]
I agree with @geaplanet, my pupils have to resize with GIMP previously the images becase shrink/grow function are unusable are they are now. In my opinion, it is nonsense because having an editor in Snap!, why go out simply to resize images?
We could use blocks to resize, but it get the code dirty unnecessarily. 
Here a little video comparing resizing with editor and with blocks.

[![resinzing](https://img.youtube.com/vi/wxb6AncW8EA/0.jpg)](https://www.youtube.com/watch?v=wxb6AncW8EA)
